scripts/single-node.sh

```diff
@@ -82,13 +82,20 @@ sed -i'.bak' 's#"null"#"kv"#g' "${CELESTIA_APP_HOME}"/config/config.toml
 # Persist ABCI responses
 sed -i'.bak' 's#discard_abci_responses = true#discard_abci_responses = false#g' "${CELESTIA_APP_HOME}"/config/config.toml
 
+# Override the log level to debug
+sed -i'.bak' 's#log_level = "info"#log_level = "debug"#g' "${CELESTIA_APP_HOME}"/config/config.toml
```

Comment:
[personal opinion][not blocking]
wouldn't this be too much for someone just wanting to run a node? debug logs are hard to follow

Reply:
I found them useful for debugging but I can understand not needing them on. Can revert.

My opinion is that this `single-node.sh` script is for developers / maintainers who are actively working on celestia-app so the configs in this file should be tailored to them. For example, tx indexing should be enabled even though it takes up more disk space b/c it's useful to query transactions.

I can see debug logs being noisy for other maintainers though so will revert

Reply:
2af00333831315af11cbb2c2a5bd2e6728bb4c0d